<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Industrial Controller Emulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <!-- Top spacer -->
      <item>
       <spacer name="verticalSpacerTopLeft">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <!-- Horizontal alignment circle -->
      <item alignment="Qt::AlignHCenter">
       <widget class="CircleWidget" name="controllerCircle">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
       </widget>
      </item>
      <!-- Horizontal alignment label -->
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="controllerStatusLabel">
       </widget>
      </item>
      <!-- Optional spacer for right label height alignment -->
      <item>
       <spacer name="labelSpacerLeft">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>15</height> <!-- Approximate height of one line of text -->
         </size>
        </property>
       </spacer>
      </item>
      <!-- Bottom spacer -->
      <item>
       <spacer name="verticalSpacerBottomLeft">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <!-- Top spacer -->
      <item>
       <spacer name="verticalSpacerTopRight">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <!-- Horizontal alignment circle -->
      <item alignment="Qt::AlignHCenter">
       <widget class="CircleWidget" name="sensorCircle">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
       </widget>
      </item>
      <!-- Horizontal alignment label -->
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="countsLabel">
       </widget>
      </item>
      <!-- Spacer снизу -->
      <item>
       <spacer name="verticalSpacerBottomRight">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CircleWidget</class>
   <extends>QWidget</extends>
   <header>CircleWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>